PerfettoCds/Pipeline/DataOutput/PerfettoProcessMemoryEvent.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Microsoft.Performance.SDK;
+using System.Collections.Generic;
+
+namespace PerfettoCds.Pipeline.DataOutput
+{
+    /// <summary>
+    /// A event that represents several memory values for a process at a point in time
+    /// </summary>
+    public readonly struct PerfettoProcessMemoryEvent
+    {
+        public string ProcessName { get; }
+        public Timestamp StartTimestamp { get; }
+        public TimestampDelta Duration { get; }
+
+        // Resident set size - anonymous memory
```

Comment:
Since these are public, maybe make these official help doc via VS /// shortcut instead of just code comment?